src/osdc/Striper.cc

```diff
@@ -524,13 +524,14 @@ uint64_t Striper::StripedReadResult::assemble_result(
     bufferlist *bl)
 {
   ldout(cct, 10) << "assemble_result(" << this << ")" << dendl;
+  uint64_t off = 0;
```

Comment:
@dillaman  How about this change in the function? 
```cpp
uint64_t Striper::StripedReadResult::assemble_result(
    CephContext *cct, std::map<uint64_t, uint64_t> *extent_map,
    bufferlist *bl)
```
Before updating, extent_map.first is image offset. Now it is the offset in the bl.